src/design/styles/base.css

```diff
@@ -29,14 +28,41 @@
 }
 
 /* Shared state styles */
-.pd-design__frame--visible {
-  outline: 1px solid var(--pd-design-color);
+.pd-design__frame.pd-design__frame--visible > .pd-design__frame--x::before,
+.pd-design__frame.pd-design__frame--visible > .pd-design__frame--x::after,
+.pd-design__frame.pd-design__frame--visible > .pd-design__frame--y::before,
+.pd-design__frame.pd-design__frame--visible > .pd-design__frame--y::after {
```

Comment:
Out of curiosity, what is pd-design__frame--x and --y? Was splitting x and y the best way to get the drop border to look good? I noticed that these drop borders go on the outside of the region whereas for composable storefronts, they were on the inside.